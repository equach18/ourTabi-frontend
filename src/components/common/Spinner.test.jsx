import { render, screen } from "@testing-library/react";
import Spinner from "./Spinner";
describe("Spinner component", () => {
  it("renders without crashing", () => {
    render(<Spinner />);
  });

  it("matches snapshot", () => {
    const { asFragment } = render(<Spinner />);
    expect(asFragment()).toMatchSnapshot();
  });
});
