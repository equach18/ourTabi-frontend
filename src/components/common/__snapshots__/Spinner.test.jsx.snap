// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Spinner component > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex justify-center items-center"
  >
    <div
      class="w-6 h-6 border-4 border-blue-500 border-t-transparent rounded-full animate-spin"
    />
  </div>
</DocumentFragment>
`;
